xattr_test.go

```diff
@@ -3,52 +3,96 @@
 package xattr
 
 import (
+	"bytes"
 	"io/ioutil"
+	"log"
 	"os"
+	"runtime"
 	"syscall"
 	"testing"
 )
 
 const UserPrefix = "user."
 
-func Test(t *testing.T) {
+// Test Get, Set, List, Remove on a regular file
+func TestRegularFile(t *testing.T) {
+	doTestRegularFile(t, false)
+}
+
+// Test LGet, LSet, LList, LRemove on a regular file
+func TestRegularFileL(t *testing.T) {
+	doTestRegularFile(t, true)
+}
+
+func doTestRegularFile(t *testing.T, followSymlinks bool) {
+	setFunc := LSet
+	setWithFlagsFunc := LSetWithFlags
+	getFunc := LGet
+	removeFunc := LRemove
+	listFunc := LList
+	if followSymlinks {
```

Comment:
maybe it's possible to create a map of test cases, e.g.:
```
cases := map[string] struct {
     fnSet,
     fnGet,
    ...,
    expected []byte
} 
```

and just iterate over:
```
for _, case := range cases {

}
```